src/sqlancer/Main.java

```diff
@@ -490,6 +523,7 @@ private boolean run(MainOptions options, ExecutorService execService,
                     DBMSExecutor<?, ?, ?> executor = executorFactory.getDBMSExecutor(databaseName, r);
                     try {
                         executor.run();
+                        System.out.println("Executor completed!");
```

Comment:
I think we should not check-in this line.

Reply:
Sorry about that, removed!